md-magic.config.js

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require("fs");
+const pathLib = require("path");
+
+const getPackageName = (path) => {
+    try {
+        const pkg = JSON.parse(fs.readFileSync(path, "utf8"));
+        return pkg.name;
+    } catch (ex) {
+        console.log(ex);
+    }
+}
+
+const getPackagePath = (path) => {
+    const inPosix = pathLib.dirname(path).split(pathLib.sep).join(pathLib.posix.sep);
+    return inPosix;
+}
+
+const getStartedInfo = (path, includeTinylicious = false) => {
+    const preamble = `<!-- The getting started instructions are automatically generated.
+To update them, edit md-magic.config.js in the root of the repo, then run npm run readme:update -->
+
+## Getting Started
+
+You can run this example using the following steps:
+
+1. Run \`npm install\` and \`npm run build:fast -- --nolint\` from the \`FluidFramework\` root directory.
+   a. For an even faster build, you can add the package name to the build command, like this:
+      \`npm run build:fast -- --nolint ${getPackageName(path)}\``;
+
+    const defaultSteps = `
+1. Navigate to this directory (${getPackagePath(path)}).
+1. Run \`npm run start\`.`;
```

Comment:
`npm run start` doesn't automatically open `localhost:8080` in the browser, so the tinylicous "In a separate terminal..." step is the only difference here.  I think changing it to something like the following to minimize the difference might be more clear/consistent:

```
const defaultSteps1 = "navigate to this directory ..."
const tinyliciousSteps = "in a separate terminal, start a tinylicious ... "
const defaultSteps2 = "run npm start from this directory and open ...."

if (includeTinylicious) {
    ...
```

Reply:
Thanks! I simplified the steps in the latest update.